develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -303,7 +309,67 @@ If a log is generated by an SDK integration, the SDK should also set the `sentry
 }
 ```
 
-Beyond these attributes, we are exploring if the SDK should also send OS, user, and device information automatically (via reading the appropriate contexts from the scope). Given this behaviour can easily be added as a new feature to the SDK, it does not have to be part of the initial SDK implementation until we make a finalized decision.
+#### User Attributes
+
+If `sendDefaultPii`/`send_default_pii` is set to `true` in the SDK, the SDK should attach the following user data if available:
+
+1. `user.id`: The user ID. Maps to `id` in the [User](/sdk/data-model/event-payloads/user/) payload.
+2. `user.name`: The username. Maps to `username` in the [User](/sdk/data-model/event-payloads/user/) payload.
+3. `user.email`: The email address. Maps to `email` in the [User](/sdk/data-model/event-payloads/user/) payload.
```

Comment:
What about the other existing attributes such as ip_address, geo, city, country_code and region? If we deliberately decided against having these, it would be good to explain here why.